code/game/objects/items/weapons/grenades/grenade.dm

```diff
@@ -64,14 +64,15 @@
 		det_time *= RAND_DECIMAL(1-variance, 1+variance)
 
 	spawn(det_time)
-		prime()
+		prime(user)
 		return
 
 
-/obj/item/weapon/grenade/proc/prime()
+/obj/item/weapon/grenade/proc/prime(mob/user as mob)
 	var/turf/T = get_turf(src)
 	if(T)
 		T.hotspot_expose(700,125)
+		user.hud_used.updatePlaneMasters(user)
```

Comment:
```suggestion
/obj/item/weapon/grenade/proc/prime(mob/user)
	var/turf/T = get_turf(src)
	T.hotspot_expose(700,125)
	user.hud_used.updatePlaneMasters(user)
```
If someone is using this in nullspace you have bigger issues so checking is pointless and it should alert you to such fallacies